hera_cal/io.py

```diff
@@ -183,14 +183,21 @@ class HERAData(UVData):
     HERAData_metas = ['ants', 'antpos', 'freqs', 'times', 'lsts', 'pols',
                       'antpairs', 'bls', 'times_by_bl', 'lsts_by_bl']
 
-    def __init__(self, input_data, filetype='uvh5'):
+    def __init__(self, input_data, filetype='uvh5', run_check=True,
```

Comment:
Why do we need these parameters? Also, if they are extremely rarely used and only by experts, perhaps we want to just collapse them into `**uvh5_read_kwargs` to avoid cluttering up the documentation?

Reply:
Ah, this was because we are accessing the `UVData.read_uvh5*` function which has these kwargs so we should really expose them. One reason is that `UVData.read_uvh5` does a check on LST values by recomputing them entirely, which for certain file shapes (e.g. long LST ranges) is actually quite intensive (i.e. ~13 seconds per file!). I agree we should use a compact `**kwargs` structure though